packages/clerk-js/src/utils/url.ts

```diff
@@ -292,6 +292,10 @@ export const generateSrc = ({ src, width }: { src?: string; width: number }) =>
     return '';
   }
 
+  if (src.substring(0, 5) === 'data:') {
```

Comment:
Maybe lets make it more generic by introducing a util

```
function isValidUrl(val?: string): val is string {
	if (!val) {
    return false
  };

	try {
		new URL(val);
		return true;
	} catch (e) {
		return false;
	}
}

```

Reply:
Check if something like that already exists though because this snippet looks familiar